src/components/tabs/js/tabsController.js

```diff
@@ -316,7 +316,11 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
       tab = elements.tabs[ i ];
       if (tab.offsetLeft + tab.offsetWidth > totalWidth) break;
     }
-    ctrl.offsetLeft = fixOffset(tab.offsetLeft);
+    if(i === elements.tabs.length - 1 && (tab.offsetLeft + tab.offsetWidth > totalWidth) && ctrl.offsetLeft === tab.offsetLeft){
```

Comment:
It checks if you're on the left side of the last tab, and there isn't enough display space to show the entire tab, in which case you need to still scroll when the user hits the right arrow.